### DISCUSSION TITLE
Pump Flow Characteristics

### DISCUSSION THREAD

**Discussion by tahirdhool** [OpenModelica/OpenModelica#11894] [General]
In the standard OpenModelica Library's example named PumpingSystem (Modelica 4.0.0+maint.om/Fluid/Examples/PumpingSystem.mo), the given flow characteristics are given as:
redeclare function flowCharacteristic = Modelica.Fluid.Machines.BaseClasses.PumpCharacteristics.quadraticFlow(V_flow_nominal = {0, 0.25, 0.5}, head_nominal = {100, 60, 0})
My question is what are these numbers: {0, 0.25, 0.5} and {100, 60, 0}? What quantities these numbers are representing and what are the units?

**Comment by casella** [comment]
The pump characteristic curve gives the head as a function of volume flow at nominal rotational speed. The curve is specified by points, which are interpolated by a polynomial (three points specify a parabola). The units are SI. Hence, there is one point a zero flow with 100 m head, one point at 0.25 m3/s at 60 m head, and one point at 0.5 m3/s at 0 m head.
